website/docs/reference/rbac.md

```diff
@@ -136,10 +136,15 @@ A user group consists of the following:
 - a **description** (optional)
 - a **list of users** (required)
 - a list of SSO groups to sync from (optional)
+- a root role associated with the group (optional) (only available in **Unleash 5.1** and later)
```

Comment:
I'm unsure about whether this should have the "(only available in ...)" or not. I'm leaning towards yes because it's unreleased for now, but I don't actually think it should stay there. So ... maybe no? What do you think?

Reply:
Yeah, I'm also in two minds. We've used this pattern in the past but I'm not 100% sure it's working for us since we apparently never go clean these things up.

That being said, I don't want users to find the docs and then ask why things don't work the way the docs say they should so while I don't like it, I think it should stay

We do seem to be pretty consistent with these version strings in the docs, though, could we script something to check this?

Reply:
Sure, I'm happy to do either, so I'll leave it up to you. 

> We do seem to be pretty consistent with these version strings in the docs, though, could we script something to check this?

Depends on what you're thinking about? We can definitely script things, but what do you want to check exactly?